### PR TITLE
Weighted percentage of known words

### PR DESCRIPTION
**Pull request by joshuagetega**
This PR introduces a getter method for computing weighted percentage of known words. Also, visually, an extra chart is displayed right below the one that shows raw percentage.

### CODE REVIEW

File: static/src/js/app/components/vocab-list-select/VocabListSelect.vue [hunk 1]
```diff
@@ -15,92 +20,92 @@
 </template>
 
 <script>
-  import { SET_VOCAB_LIST } from '../../constants';
-  import VocabListEntry from './VocabListEntry.vue';
-  import VocabListSelectedEntry from './VocabListSelectedEntry.vue';
+import { SET_VOCAB_LIST } from "../../constants";
```
Comment (paltman) [inline static/src/js/app/components/vocab-list-select/VocabListSelect.vue]: Let's keep the indentation and single-quotes within Vue/JavaScript, please.

**Comment by joshuagetega** [comment]
Sure thing, @paltman. I'll fix and let you know.

File: static/src/js/app/components/vocab-list-select/VocabListSelect.vue (hunk 1 above)
Comment (joshuagetega) [reply static/src/js/app/components/vocab-list-select/VocabListSelect.vue]: Done.

**Comment by joshuagetega** [comment]
@paltman, I have removed the format changes. The PR should be in order now. Please do not hesitate to let me know of any additional changes to make.

**Pull request merged**